src/ode_solver/runge_kutta_methods/rk_tableau_base.h

```diff
@@ -38,6 +37,15 @@ class RKTableauBase
 
     /// Butcher tableau "b"
     dealii::Table<1,double> butcher_tableau_b;
+    
+    /// Setter for butcher tableau
+    virtual void set_tableau() = 0;
```

Comment:
If it's definition is always the same, define it in the base class

Reply:
Changed such that `set_tableau()` is defined in the base class, according to our discussion. This change means that `RungeKuttaODESolver::allocate_ode_system()` now has to call `butcher_tableau->set_tableau()`.